GITHUB_PROJECTS_SETUP.md

```diff
@@ -0,0 +1,100 @@
+# GitHub Projects Setup
+
+This article describes how we manage [our product](https://github.com/orgs/SRGSSR/projects/9) with [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).
+
+## Settings
+
+We currently use 3 settings:
+
+- **Status** for the various statuses an issue might have in our worklow. Statuses must be defined in the following order so that board columns are ordered correctly:
```

Comment:
```suggestion
- **Status** for the various statuses an issue might have in our workflow. Statuses must be defined in the following order so that board columns are ordered correctly:
```